src/stores/pacbioPoolCreate.js

```diff
@@ -916,7 +1032,24 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
 
     selectRequest({ id, selected = true }) {
       if (selected) {
-        this.used_aliquots[`_${id}`] = createUsedAliquot({ source_id: id })
+        //If the libraries contain the request, fill the used_aliquot values with the library attributes values for template_prep_kit_box_barcode, volume, concentration, and insert_size
+        const library = Object.values(this.resources.libraries).find(
```

Comment:
If libraries are indexed by ID we can do `this.resources.libraries[library.request]`

Reply:
Yes, upon checking, it seems the library's ID is different from library.request. So, we might not be able to access it this way, which could result in retrieving the wrong library object.

Reply:
Yes, but on check it seems libraries id is different from library.request. So, we might not be able to access it this way which might return a wrong library object

Reply:
As yes its not the library ID we are using.